Program1.java

```diff
@@ -0,0 +1,40 @@
+public class Program1 {
```

Comment:
```suggestion
public class PrimeRangeNoLoop {
```
Consider naming the class in such a way that it indicates the function of the class.